src/components/activities.js

```diff
@@ -314,7 +317,24 @@ export class ActivityPorts {
     if (addDefaultArguments) {
       args = this.addDefaultArguments(args);
     }
-    return this.openActivityIframePort_(iframe, url, args);
+    return this.deps_
+      .storage()
+      .get(Constants.USER_TOKEN, true)
+      .then((swgUserToken) => {
+        const split = url.split('?');
```

Comment:
I'm pretty sure you don't need to do this and can just do

```javascript
const queryParams = new URL(url).searchParams;
if (swgUserToken && !queryParams.has('sut')) {
    // stuffs
}
```

Reply:
Done. URL() is picky about the input, so I had to change `/hello` to `https://www.google.com/` in the tests. Don't think that will be an issue, but wanted to bring that up.